src/wallet/rpc/spend.cpp

```diff
@@ -983,6 +1003,11 @@ static RPCHelpMan bumpfee_helper(std::string method_name)
                              "are replaceable).\n"},
                     {"estimate_mode", RPCArg::Type::STR, RPCArg::Default{"unset"}, "The fee estimate mode, must be one of (case insensitive):\n"
                              "\"" + FeeModes("\"\n\"") + "\""},
+                    {"extra_outputs", RPCArg::Type::ARR, RPCArg::Default{UniValue::VARR}, "Additional outputs (key-value pairs) in the same format\n"
+                             "as the \"outputs\" argument to \"send\": each address can only appear once and there can only be\n"
+                             "one \"data\" object.\n",
+                         OutputsDoc(),
+                     },
```

Comment:
```suggestion
                    },
```
nit: misaligned brackets

Reply:
done